avocado/core/sysinfo.py

```diff
@@ -238,7 +238,7 @@ def _get_cursor(self):
         try:
             cmd = 'journalctl --quiet --lines 1 --output json'
             result = process.system_output(cmd, verbose=False)
-            last_record = json.loads(result)
+            last_record = json.loads(astring.to_text(result))
```

Comment:
`result.stdout_text` is preferred

Reply:
The prequiste for result.stdout_text is that result should be cmdResult, but here result is just output,which is either bytes or string